### ISSUE TITLE
Market research | Playgroups, activities offered and pricing

### ISSUE THREAD

**Issue by iteles** [bmhco/shared#83]
@Stephanymtr As we discussed last week, could you please do some research so that we can better understand the expectations that have already been set for playgroups in Portugal?

We're looking for:
- Playgroup name
- Link to info
- Location
- Duration
- Times offered (maybe in a spreadsheet we could have 3 columns with either morning/afternoon/all day and then you just fill in the times in the relevant column so we can also see what the most popular offerings are)
- Days of the week (fixed vs floating days)
- Activities offered
- Parent supervision mandatory? Is parent supervision even possible? Can you leave the kids there without parents?
- Price for each duration (if relevant)

Feel free to also open an issue where you can quickly drop any cool ideas you see for activities they are doing as well please 🙏

**Comment by Stephanymtr** [comment]
I've just shared the file with you @iteles. Here is the link for easy access https://docs.google.com/spreadsheets/d/1q7fRZVpX9xBWOGdJtA2xR7vanzHitFWs_Bp0WERYe7c/edit

**Comment by iteles** [comment]
@Stephanymtr Thank you for this!
Are you continuously adding to it at nap times and spreading the research throughout the rest of Portugal?
Lisbon and surrounding areas probably also has a more structured approach.

**Comment by Stephanymtr** [comment]
Yes and yes, the list is not done yet. There are still lots of playgroups to add. 

I will check out and add the ones I find from Lisbon :)

**Comment by iteles** [comment]
Thanks @Stephanymtr :) Please also remember to note down any cool activity ideas you see with illustrative pictures in a separate issue 🙏

**Comment by Stephanymtr** [comment]
Sure thing! 

https://github.com/bmhco/shared/issues/84
